apps/evm/src/clients/api/queries/useGetPools/getPools/getApiPools/index.ts

```diff
@@ -10,7 +10,11 @@ export interface ApiRewardDistributor {
   supplySpeed: string;
   borrowSpeed: string;
   priceMantissa: string;
-  rewardsDistributorContractAddress: string | null;
+  rewardsDistributorContractAddress: string;
+  rewardType: 'venus_isolated_market' | 'venus_legacy_market' | 'merkl';
+  rewardDetails?: {
+    description?: string;
+  };
```

Comment:
Since you've added a `rewardType` property already, I'd recommend adding a `rewardDescription` instead of an object with just one property here.